doc/hardware/porting/board_porting.rst

```diff
@@ -956,3 +956,23 @@ extension.
    Board extensions are not allowed in any module referenced in Zephyr's
    ``west.yml`` manifest file. Any board changes are required to be submitted to
    the main Zephyr repository.
+
+.. _board_terminology:
+
+Terminology
```

Comment:
would be nicer if you used some formatting for this type of thing, so it is easy to read and find:

https://sublime-and-sphinx-guide.readthedocs.io/en/latest/glossary.html#create-a-glossary

Reply:
would be nice as a followup to have few diagrams showing all of this and the layering and how things relate and what type of "setups" are possible. a picture is worth 1000 words....

Reply:
> would be nicer if you used some formatting for this type of thing, so it is easy to read and find:
> 
> https://sublime-and-sphinx-guide.readthedocs.io/en/latest/glossary.html#create-a-glossary

Thanks for the link, will take a look at updating.
I just followed what we do here: https://docs.zephyrproject.org/latest/project/index.html#terminology

Perhaps long term we should try to have a uniform terminology style, fyi @kartben

Reply:
now that you mention this, you should remove the "/" from identifier :)

Reply:
and yes, we can start with a link to this table.

Reply:
> > would be nicer if you used some formatting for this type of thing, so it is easy to read and find:
> > https://sublime-and-sphinx-guide.readthedocs.io/en/latest/glossary.html#create-a-glossary
> 
> Thanks for the link, will take a look at updating.

Done

Reply:
> and yes, we can start with a link to this table.

added a reference to the section as I couldn't get referencing the table directly to work.

Reply:
> now that you mention this, you should remove the "/" from identifier :)

done